### PR TITLE
feat: add honeycomb changes

### PR DESCRIPTION
**Pull request by suivo2001**
Update electron env vars and add config object.

### CODE REVIEW

File: src/config/main.js [hunk 1]
```diff
@@ -5,7 +5,8 @@
 import { jsPsych } from "jspsych-react";
 import _ from "lodash";
 import { eventCodes } from "./trigger";
-import { verifyProlific } from "../lib/utils";
+import { init } from "@brown-ccv/behavioral-task-trials";
```
Comment (mcmcgrath13) [inline src/config/main.js:8]: do we really need the behavioral task trials? it doesn't look like we're using any of the trials, so the init won't have an impact

File: src/config/main.js [hunk 2]
```diff
@@ -25,54 +26,57 @@ const audioCodes = {
...
+if (!USE_ELECTRON) {
   // if this is mturk, merge in the mturk specific language
   const mlang = require("../language/en_us.mturk.json");
   _.merge(lang, mlang);
 }
 
 const taskName = "MSIT";
 
+// setting config for trials
+const config = init({
```
Comment (mcmcgrath13) [inline src/config/main.js:64]: if you aren't using behavioral task trials at all, can just have config equal the object

**Review by mcmcgrath13 — COMMENTED** [review]
awesome! just a couple questions

File: src/config/main.js (hunk 2 above)
Comment (suivo2001) [reply src/config/main.js:64]: Yes that makes sense.

**Review by mcmcgrath13 — APPROVED** [review]
🔥

**Pull request merged**